src-tauri/src/lib.rs

```diff
@@ -471,3 +477,37 @@ pub fn set_mod_enabled_status(
 
     Ok(())
 }
+
+/// Gets list of installed mods and their properties
+/// - name
+/// - is enabled?
+pub fn get_installed_mods(game_install: GameInstall) -> Result<Vec<NorthstarMod>, String> {
+    let enabled_mods_json_path = format!("{}/R2Northstar/enabledmods.json", game_install.game_path);
+    // Open file
+    let data = match std::fs::read_to_string(enabled_mods_json_path) {
+        Ok(data) => data,
+        Err(err) => return Err(err.to_string()),
+    };
+    // Check if valid JSON and parse
+    let res: serde_json::Value = match serde_json::from_str(&data) {
+        Ok(res) => res,
+        Err(err) => return Err(err.to_string()),
+    };
+
+    let mut installed_mods = Vec::new();
+
+    for (key, value) in res.as_object().unwrap() {
+        println!("{:?} ===> {:?}", key, value);
```

Comment:
You should remove this.

Reply:
Done with 6148dd0945786b93473b5f58201b0671bcd2ec96